src/common/applicationDetails/ApplicationDetails.tsx

```diff
@@ -176,30 +185,40 @@ const ApplicationDetails = ({
             )}
           </Section>
         ) : (
-          <Section title={t('applicationList.applicationDetails.selectedPorts')}>
-            <List noBullets>
-              {[...harborChoices]
-                .filter(notNull)
-                .sort((choiceA, choiceB) => choiceA.priority - choiceB.priority)
-                .map(({ harborName, harbor }, i) => {
-                  routerQuery.set('harbor', harbor);
+          choices.length > 0 && (
```

Comment:
This choice list could be separated into a component

Reply:
Done.